composer.json

```diff
@@ -57,6 +57,6 @@
             "dev-master": "2.x-dev"
         }
     },
-    "minimum-stability": "dev",
+    "minimum-stability": "RC",
```

Comment:
Can you keep this as dev (makes it easier to start testing on dev branches before RC)?

Reply:
My plan was to remove this completly next week. After the first stable release of symfony 7.
Otherwise it installs symfony 7.1-dev which of course could have other bugs.

Reply:
It won't install Symfony 7.1 until we add to the matrix (prefer-stable prevents this).

Reply:
You are right for ci.
But it installs 7.1 on my local test environment. Sorry i did not make that clear in my comment.

Reply:
> But it installs 7.1 on my local test environment.

That's weird, just with a standard `composer update`?

Reply:
I test it local with this to get 7.0:
```
"minimum-stability": "RC",
"prefer-stable": false
```

Gets me 7.1 local.
```
"minimum-stability": "dev",
"prefer-stable": false
```

Gets me 6.3 local.
```
"minimum-stability": "dev",
"prefer-stable": true
```

I changed it now to `dev`. CI is more important than my local temp environment. ;)

Reply:
aah, ok, I understand now. Here's the method I use:

1. `composer global install symfony/flex`
2. `SYMFONY_REQUIRE=6.4.* composer update` (test)
3. `SYMFONY_REQUIRE=7.0.* composer update` (test)
4. `SYMFONY_REQUIRE=7.1.* composer update` (test)
5. ...

Reply:
Thanks for sharing. I will try this next time